modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/GmosEpics.scala

```diff
@@ -22,7 +24,7 @@ class GmosEpics(epicsService: CaService) {
     val stageMode = cs.map(_.getString("stageMode"))
     def setStageMode(v: String): SeqAction[Unit] = setParameter(stageMode, v)
 
-    val useElectronicOffsetting = cs.map(_.addInteger("useElectronicOffsetting", "wfs:followA.K", "Enable electronic Offsets", false))
+    val useElectronicOffsetting = cs.map(_.addInteger("useElectronicOffsetting", GMOS_TOP + "wfs:followA.K", "Enable electronic Offsets", false))
```

Comment:
The other vals call getXXX, which retrieves an already existing parameter. `useElectronicOffsetting` is special: it cannot be build from the XML file. Here it is being built in the call to addInteger. I think the confusion is that this one includes the EPICS channel name in its parameters, which was the one missing the prefix.